src/strava_gear/input/activities.py

```diff
@@ -47,8 +47,8 @@ def read_strava_offline(db_filename: Union[str, PathLike]) -> Tuple[Dict[BikeNam
         }
 
         activities: List[Dict] = []
-        for r in db.execute('SELECT name, gear_id, start_date, moving_time, distance FROM activity'):
-            assert set(r.keys()) <= {'name', 'gear_id', 'start_date', 'moving_time', 'distance'}
+        for r in db.execute('SELECT name, gear_id, type, start_date, moving_time, distance FROM activity'):
+            assert set(r.keys()) <= {'name', 'gear_id', 'type', 'start_date', 'moving_time', 'distance'}
```

Comment:
Can you update the other input method (csv) as well?

(I should've had tests for that one, sorry about that!)

Reply:
Ah I missed it. Fixed in the latest commit.